_includes/v22.2/misc/session-vars.md

```diff
@@ -3,6 +3,7 @@
 | <a name="application-name"></a> `application_name` | The current application name for statistics collection. | Empty string, or `cockroach` for sessions from the [built-in SQL client](cockroach-sql.html). | Yes | Yes |
 | <a name="bytea-output"></a> `bytea_output` | The [mode for conversions from `STRING` to `BYTES`](bytes.html#supported-conversions). | hex | Yes | Yes |
 | <a name="client-min-messages"></a> `client_min_messages` | The severity level of notices displayed in the [SQL shell](cockroach-sql.html). Accepted values include `debug5`, `debug4`, `debug3`, `debug2`, `debug1`, `log`, `notice`, `warning`, and `error`. | `notice` | Yes | Yes |
+| <a name="copy-from-atomic-enabled"></a> `copy_from_atomic_enabled` | **New in v22.2:** If set to `on`, [`COPY FROM`](copy-from.html) statements are committed atomically, matching PostgreSQL behavior. If set to `off`, `COPY FROM` statements are segmented into batches of 100 rows unless issued within an explicit transaction, matching the behavior in versions prior to v22.2. | `on` | Yes | Yes |
```

Comment:
Total nit: But since you mention "PostgreSQL behavior" in the first sentence and then in the second sentence "matching the behavior in versions prior to v22.2", I kind of did a double take of whether we meant CRDB or PostgreSQL's version. Maybe it's just me! Though a "prior to CockroachDB v22.2" might clarify.